src/Support/Environment.php

```diff
@@ -13,6 +14,6 @@ final class Environment
 {
     public static function isALaravelApplication(): bool
     {
-        return class_exists('\App\Http\Kernel') && class_exists(ParallelRunner::class);
+        return class_exists(Application::class) && !class_exists(TestCase::class);
```

Comment:
How about "class_exists(ParallelRunner::class)" too?

Reply:
In the `composer.json` we have
```json
"conflict": {
    "laravel/framework": "<8.55"
},
``` 
and Laravel 8.55 and above has `ParallelRunner` so verifying that `laravel/framework` is installed should be enough.

If you want, I can modify this line to 
```php
return class_exists(Application::class) && class_exists(ParallelRunner::class) && !class_exists(TestCase::class);
```

Reply:
@nunomaduro What do you think about that ?